content/projects/gsoc/2020/projects/external-fingerprint-storage.adoc

```diff
@@ -15,81 +15,263 @@ links:
   gitter: jenkinsci/external-fingerprint-storage
   draft: https://docs.google.com/document/d/10f3IXTA6UMLUOFMTH_atQ3XlyWB3S7KGNCtTZmOUGdM/edit?usp=sharing
   idea: /projects/gsoc/2020/project-ideas/external-fingerprint-storage-for-jenkins
-  meetings: "/projects/gsoc/2020/projects/external-fingerprint-storage#meetings"
 tags:
 - gsoc2020
 - pluggable-storage
 - fingerprints
+- Redis
+- PostgreSQL
 ---
 
 === Abstract
 
-File fingerprinting is a way to track which version of a file is being used by a job/build, making dependency tracking easy.
-The fingerprint engine of Jenkins can track usages of artifacts, credentials, files, etc. within the system.
+File fingerprinting is a way to track which version of a file is being used by a job/build, making dependency tracking
+easy.
+The fingerprinting engine of Jenkins can track usages of artifacts, credentials, files, etc. within the system.
 It does this by maintaining a local XML-based database.
-This leads to dependence on the physical disk of the Jenkins master.
+This led to dependence on the physical disk of the Jenkins master.
 
-The core idea of this project is to extend Jenkins core to support storing of fingerprints in an external storage, which would also allow tracking them across the entire CI/CD flow.
+This project extended Jenkins core to support storing of fingerprints in external storages.
+External storage plugins can then be used to configure connection to the storage.
+Once configured, the fingerprints are stored and loaded by Jenkins core via these plugins in a seamless manner.
+The project led to the creation of two plugins, backed by Redis and PostgreSQL.
```

Comment:
You could also embed a demo or a slidedeck here so that visitor could browse through the overview

Reply:
I think it makes sense to add the final slides with work of all phases rather than a single phase. I can update that later, for the time being I have added link to demo :)